code/modules/antagonists/cult/runes.dm

```diff
@@ -309,6 +309,29 @@ structure_check() searches for nearby cultist structures required for the invoca
 				to_chat(M, "<span class='cultlarge'>\"I accept this sacrifice.\"</span>")
 			else
 				to_chat(M, "<span class='cultlarge'>\"I accept this meager sacrifice.\"</span>")
+	
+	if(iscyborg(sacrificial))
+		var/static/list/constructs = list(
+			"Juggernaut" = image(icon = 'icons/mob/cult.dmi', icon_state = "juggernaut"),
+			"Wraith" = image(icon = 'icons/mob/cult.dmi', icon_state = "wraith"),
+			"Artificer" = image(icon = 'icons/mob/cult.dmi', icon_state = "artificer")
+			)
```

Comment:
We 100% have this exact radial when making a shell or whatever, make it a list that both can access.